casper/src/main/scala/coop/rchain/casper/engine/NodeLaunch.scala

```diff
@@ -196,6 +197,7 @@ object NodeLaunch {
       quarantineLength: Int,
       numberOfActiveValidators: Int,
       posMultiSigPublicKeys: List[String],
+      posVaultPubKey: String,
```

Comment:
Keep `posMultiSigPublicKeys` and `posMultiSigQuorum` parameters together. They are part of the same functionality.